x-pack/platform/plugins/shared/onechat/public/application/components/conversations/conversation_rounds/round_layout.tsx

```diff
@@ -8,13 +8,23 @@
 import { EuiFlexGroup, EuiFlexItem, EuiPanel, useEuiTheme } from '@elastic/eui';
 import { css } from '@emotion/react';
 import type { ReactNode } from 'react';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { i18n } from '@kbn/i18n';
 
+// A new round should have a min-height equal to the scroll container height
+const getScrollContainerHeight = () => {
```

Comment:
are we able to adjust this to use a ref instead?

Reply:
Yes, definitely possible and cleaner. We'll just need to pass the ref (or just the height value) down 2 levels from `conversation.tsx` --> `conversation_rounds.tsx` --> `<RoundLayout ... />` - given this, is it still your preferred approach?

Reply:
Nice suggestion! This is a better approach https://github.com/elastic/kibana/pull/234367/commits/4e359af54bcd61162fd04293980541457cd7820a -- I also cleaned up the unnecessary div and the ref being on the wrong element.